crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -493,7 +492,17 @@ where
         };
 
         if !no_info {
-            print_disk_farm_info(disk_farm.directory, disk_farm_index);
+            let info = single_disk_farm.info();
+            println!("Single disk farm {disk_farm_index}:");
```

Comment:
Does this not duplicate the message logic somewhere?

Reply:
A bit, there is a dedicated `info` command, but that command also handles errors in case farm does not yet exists, so I thought copying it is fine.